docs/csharp/quick-starts/branches-and-loops-local.md

```diff
@@ -0,0 +1,305 @@
+
+```csharp
+ int c = 4;
+ if ((a + b + c > 10) && (a > b))
+{
+    Console.WriteLine("The answer is greater than 10");
+    Console.WriteLine("And the first number is greater than the second");
+}
+else
+{
+    Console.WriteLine("The answer is not greater than 10");
+    Console.WriteLine("Or the first number is not greater than the second");
+}
+```
+
+The `&&` represents "and". It means both conditions must be true to execute
+the statement in the true branch.  These examples also show that you can have multiple
+statements in each conditional branch, provided you enclose them in `{` and `}`.
+
+You can also use  `||` to represent "or". Add the following code after what you've written so far:
+
+```csharp
+if ((a + b + c > 10) || (a > b))
+{
+    Console.WriteLine("The answer is greater than 10");
+    Console.WriteLine("Or the first number is greater than the second");
+}
+else
+{
+    Console.WriteLine("The answer is not greater than 10");
+    Console.WriteLine("And the first number is not greater than the second");
+}
+```
+
+You've finished the first step. Before you start the next section, let's move the current code into a separate method. That makes it easier to start working with a new example. Rename your `Main` method to `ExploreIf` and write a new `Main` method that calls `ExploreIf`. When you have finished, your code should look like this:
+
+```csharp
+using System;
+
+namespace BranchesAndLoops
+{
+    class Program
+    {
+        static void ExploreIf()
+        {
+            int a = 5;
+            int b = 3;
+            if (a + b > 10)
+            {
+                Console.WriteLine("The answer is greater than 10");
+            }
+            else
+            {
+                Console.WriteLine("The answer is not greater than 10");
+            }
+
+            if ((a + b + c > 10) && (a > b))
+            {
+                Console.WriteLine("The answer is greater than 10");
+                Console.WriteLine("And the first number is greater than the second");
+            }
+            else
+            {
+                Console.WriteLine("The answer is not greater than 10");
+                Console.WriteLine("Or the first number is not greater than the second");
+            }
+
+            if ((a + b + c > 10) || (a > b))
+            {
+                Console.WriteLine("The answer is greater than 10");
+                Console.WriteLine("Or the first number is greater than the second");
+            }
+            else
+            {
+                Console.WriteLine("The answer is not greater than 10");
+                Console.WriteLine("And the first number is not greater than the second");
+            }            
+        }
+
+        static void Main(string[] args)
+        {
+            ExploreIf();
+        }
+    }
+}
+```
+
+## Use loops to repeat operations
+
+Comment out the call to `ExploreIf()`. It will make the output less cluttered as you work in this section:
```

Comment:
I would move all the non-loop info to the preceding section. Anyone jumping to this H2 will be confused.